### ISSUE TITLE
MMD Loss问题

### ISSUE THREAD

**Issue by ChangLiu2024** [ArthurWish/DCTLN-#1]
为什么MMD Loss foward最后要加torch.no_grad()？这样MMD岂不是不会更新模型了？
